template/dockerfiles/csharp/draft.yaml

```diff
@@ -1,11 +1,13 @@
 language: csharp
+displayName: C#
 variables:
   - name: "PORT"
     description: "the port exposed in the application"
     type: int
   - name: "VERSION"
     description: "the dotnet SDK version"
     type: float
+    exampleValues: ["3.1","4.0","5.0","6.0"]
```

Comment:
super nit: some example values are ascending and some are descending, would we be able to standardize order?

(personal preference is for ascending but don't think it matters as long as it's consistent)

Reply:
np i went with ascending